tests/rest/datacube/test_datacube.py

```diff
@@ -102,6 +102,19 @@ def test_filter_temporal_basic_extent(s2cube):
     assert graph['arguments']['extent'] == ["2016-01-01", "2016-03-10"]
 
 
+def test_load_stac_connectionless(connection):
+    expected_graph = {
+        "loadstac1": {
+            "process_id": "load_stac",
+            "arguments": {"url": "https://provider.test/dataset"},
+            "result": True,
+        }
+    }
+    cube = DataCube.load_stac("https://provider.test/dataset")
+    assert cube.flat_graph() == expected_graph
+    cube2 = connection.load_stac("https://provider.test/dataset")
+    assert cube2.flat_graph() == expected_graph
```

Comment:
Can you move this  test to this class https://github.com/Open-EO/openeo-python-client/blob/3871077d53a4f2f2b7addc327c6a15fef4a45ab2/tests/rest/datacube/test_datacube.py#L85-L136
which I just added with 3871077d53a4f2f2b7addc327c6a15fef4a45ab2

Reply:
Moved!